CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Unreleased
     - Front end improvements:
         - Highlight pin on sidebar focus as well as hover.
+        - /around pagination links now styled as links rather than buttons.
```

Comment:
Technically this also affects `/reports` and `/my` too.

Reply:
Good point! Will amend the commit message and changlog entry.